addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1017,6 +1030,7 @@ def channel_fetched(self):
                 'channel_id': channel.id,
                 'id': member.id,
                 'last_message_id': last_message_id,
+                'member_id': member.id,
                 'partner_id': self.env.user.partner_id.id,
```

Comment:
Don't we want to handle guests here too?

Reply:
I believe we should extend the channel fetched support in master if we really want to: the route is not even created.